tests/test_database.py

```diff
@@ -767,7 +767,26 @@ def test_the_homer(self):
         self.assertEqual(tank_temperature.choices, None)
         self.assertEqual(tank_temperature.comment, None)
 
-        speed_km = db.messages[1].signals[1]
+        info_0 = db.messages[1].signals[0]
+
+        self.assertEqual(info_0.name, 'Info0')
+        self.assertEqual(info_0.start, 0)
+        self.assertEqual(info_0.length, 8)
+        self.assertEqual(info_0.receivers, [])
+        self.assertEqual(info_0.byte_order, 'little_endian')
+        self.assertEqual(info_0.is_signed, False)
+        self.assertEqual(info_0.is_float, False)
+
+        abs_info_mux = db.messages[1].signals[8]
+        self.assertEqual(abs_info_mux.name, 'ABS_InfoMux')
+        self.assertEqual(abs_info_mux.start, 16)
+        self.assertEqual(abs_info_mux.length, 2)
+        self.assertEqual(abs_info_mux.receivers, [])
+        self.assertEqual(abs_info_mux.byte_order, 'little_endian')
+        self.assertEqual(abs_info_mux.is_signed, False)
+        self.assertEqual(abs_info_mux.is_float, False)
```

Comment:
Verify the multiplexer attributes as well, as those are an important part of the new functionality.

Reply:
Good point. I have added it there.